### PR TITLE
Stable jkphl/dom-factory support

### PR DESCRIPTION
**Pull request by blankse**
micrometa only support dom-factory versions < 1 now. Should also support the stable versions 1 of dom-factory

### CODE REVIEW

**Comment by blankse** [comment]
@rvanlaak I still can't install micrometa with PHP8 and Guzzle 7 :( Can you review this and https://github.com/jkphl/rdfa-lite-microdata/pull/9 ?

**Comment by blankse** [comment]
@rvanlaak Can you here also make a new release with this?

File: composer.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
     },
     "require": {
         "php": ">=7.1.3",
-        "jkphl/dom-factory": "^0.1.2",
+        "jkphl/dom-factory": "^0.1.2 || ^1",
```
Comment (rvanlaak) [inline composer.json:26]:
```suggestion
        "jkphl/dom-factory": "^1",
```

**Comment by rvanlaak** [comment]
tagged v3.1.1

**Pull request merged**